Fix the bug that `<limits>` is missing in versions less than 7.0.5 (https://github.com/jtv/libpqxx/commit/143e139fafadfb153a5d0c84c429e6ced3ed06fc)
diff --git a/include/pqxx/util.hxx b/include/pqxx/util.hxx
index d91d978e..c589a42c 100644
--- a/include/pqxx/util.hxx
+++ b/include/pqxx/util.hxx
@@ -17,6 +17,7 @@
 #include <cctype>
 #include <cstdio>
 #include <iterator>
+#include <limits>
 #include <memory>
 #include <stdexcept>
 #include <string>
